pennylane/optimize/qng.py

```diff
@@ -23,15 +23,15 @@
 
 class QNGOptimizer(GradientDescentOptimizer):
     r"""Optimizer with adaptive learning rate, via calculation
-    of the quantum geometric tensor or Fubini-Study metric tensor.
+    of the Fubini-Study metric tensor.

```

Comment:
how come you modified these?

Reply:
On re-reading this docstring, I realised it was a bit confusing, and didn't match the notation in the paper 100%. In fact, I think it's still incorrect, it should specify block-diagonal/diagonal.
```suggestion
    of the diagonal or block-diagonal approximation to the Fubini-Study metric tensor.
```